pytests/func/test_breakpoints.py

```diff
@@ -126,3 +127,134 @@ def code_to_debug():
             session.wait_for_thread_stopped()
             session.send_request('continue').wait_for_response(freeze=False)
         session.wait_for_exit()
+
+
+def test_crossfile_breakpoint(pyfile, run_as, start_method):
+    @pyfile
+    def script1():
+        # import_and_enable_debugger()
```

Comment:
Might as well import it twice - it doesn't hurt, since the second import will be  a no-op.